### PR TITLE
[feature/bootstrap-vue] Add bootsrap-vue

### PR DESCRIPTION
**Pull request by sskorupski**
This is the PR (Pull Request) from  the bootstrap-vue integration.

I tried to keep the global look & feel but the paper  theme used was in conflicts with bootstrap css.
So I started to remove some of the scss paper theme import.
From this point we loose the navbar behaviour on resolution below 991px. I considered that as far as the project is a starter kit, that was not a so big issue in comparison of the added bootstrap-vue added features.

If we want to get back the sidebar loose feature I will suggest to try to make it back in another PR (with a low priority)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,10 +8,12 @@
     "clean": "rimraf dist",
     "start": "webpack-dev-server --hot --inline --progress",
     "build": "webpack --progress",
-    "prod": "npm run clean && webpack -p --progress"
+    "prod": "npm run clean && webpack -p --progress",
+    "winstop": "Taskkill /IM node.exe /F"
```
Comment (sskorupski) [inline package.json:12]: As far as I develop on a windows OS, I added this script to help to kill all the node running process that still run after `ctrl + c`  usage command in the console

File: package.json [hunk 2]
```diff
@@ -37,6 +39,7 @@
     "friendly-errors-webpack-plugin": "^1.1.3",
     "html-webpack-plugin": "^2.28.0",
     "http-proxy-middleware": "^0.17.3",
+    "jquery": "^3.3.1",
```
Comment (sskorupski) [inline package.json:42]: jquery is needed to make bootsrap-vue works properly

File: package.json [hunk 3]
```diff
@@ -8,10 +8,12 @@
     "clean": "rimraf dist",
     "start": "webpack-dev-server --hot --inline --progress",
     "build": "webpack --progress",
-    "prod": "npm run clean && webpack -p --progress"
+    "prod": "npm run clean && webpack -p --progress",
+    "winstop": "Taskkill /IM node.exe /F"
   },
   "dependencies": {
-    "bootstrap": "^3.3.7",
+    "ajv": "^6.9.2",
```
Comment (sskorupski) [inline package.json:15]: ajv is a required jquery not embedded dependency that is needed

File: package.json [hunk 4]
```diff
@@ -8,10 +8,12 @@
...
+    "bootstrap-vue": "^2.0.0-rc.13",
```
Comment (micobarac) [inline package.json:16]: We should consider updating package.json dependencies, as the ones inside are pretty old.

**Pull request merged**